plugin/src/main/kotlin/de/triplet/gradle/play/BootstrapTask.kt

```diff
@@ -86,4 +86,4 @@ open class BootstrapTask : PlayPublishTaskBase() {
 
     private fun String.write(dir: File, detail: ListingDetail) =
             File(dir, detail.fileName).writeText(this)
-}
+}
```

Comment:
@gtcompscientist Lol, now we don't have a new line. Can we add 1 back here and in `Tracks.kt`. I recommend going into IntelliJ/AS `Settings -> Editor -> General`. Under `Other`, you'll find `Ensure line feed at file end on save`.

Reply:
Hahaha, yeah.